commons/src/test/java/io/aiven/kafka/connect/common/source/OffsetManagerTest.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2024 Aiven Oy
```

Comment:
```suggestion
 * Copyright 2025 Aiven Oy
```
introduced in this year